libraries/architecture/build.gradle.kts

```diff
@@ -18,16 +18,16 @@
 @Suppress("DSL_SCOPE_VIOLATION")
 plugins {
     id("io.element.android-compose-library")
-    alias(libs.plugins.molecule)
 }
 
 android {
-    namespace = "io.element.android.x.architecture"
+    namespace = "io.element.android.x.libraries.architecture"
```

Comment:
we should change all the other `libraries` namespaces then.

Reply:
Oh I thought it was already the case. What do you prefer?

Reply:
not a strong pref, the risk of name collision is low even if we do not include `libraries`. But maybe for coherency keep the same path than the module path.

Reply:
will do a clean-up in a separate PR as there are some mix between package name, namespace and module hierarchy.